Remove duplicates from 12 factors

Charchit26 opened the pull request:
Added `Codebase`, Dependencies`, `Config`, Backing Services` to the 12 factors pointed out.

Charchit26 committed:
Remove duplicates from 12 factors

Added `Codebase`, Dependencies`, `Config`, Backing Services` to the 12 factors pointed out.

suryaval reviewed (changes requested):
Hi @Charchit26 
Please see the 12 factor app principles here:
https://12factor.net

Please elaborate why this change is required.

Charchit26 commented:
Hi @suryaval 
Your list of 12 factor app principles contains redundant items like `process` and `disposability`. Whereas it does not have principles like `codebase` which actually exists in 12 factors. I have just removed the duplicates and updated with the missing items.
Kindly double check the list of the principles.
![capture](https://user-images.githubusercontent.com/19488985/48969322-2cbe7100-f023-11e8-97b1-7ce0692cdeb0.PNG)

Thanks

suryaval commented:
@Charchit26 Thanks for the elaboration. The subset i saw was on a fork.
